src/sass/app_2/_common/form/input_field.scss

```diff
@@ -156,7 +157,8 @@
         &:active, &:focus {
             outline: 0;
             border-radius: $BORDER_RADIUS;
-            border-color: $COLOR_BLUE;
+            border-color: $COLOR_WHITE;
+            box-shadow: 0 0 0 0.2rem rgba(127, 131, 151, 0.3) // stylelint-disable-line number-leading-zero
```

Comment:
Do we still need the `stylelint-disable-line` for this rule? For `number-leading-zero` seems like there are no rules violations here. It's only a violation if you use `.3` instead of `0.3`.

Reply:
its because the box-shadow start with 0 from `0 0 0 0.2rem`

Reply:
That's strange, does it not pass the rule if you remove it?